task2/main.c

```diff
@@ -0,0 +1,79 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+#include "graph.h"
+#include "algorithm.h"
+
+#define TEST_FAILED -1
+
+typedef enum
+{
+    ok,
+    loadError,
+    outOfMemory,
+    openFileError
+} ErrorCodes;
+
+bool isEqual(const int** result, const size_t numberOfComponents)
+{
+    const int expected[4][4] = { {0,1,0,0},
```

Comment:
```suggestion
    const int expected[4][4] = { {0, 1, 0, 0},
```
И ниже. После запятой всегда ставится пробел, даже если очень не хочется